core/src/main/java/com/linecorp/armeria/server/docs/DocServiceBuilderSupplier.java

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2019 LINE Corporation
+
+LINE Corporation licenses this file to you under the Apache License,
+version 2.0 (the "License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at:
+
+  https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+License for the specific language governing permissions and limitations
+under the License.
+*/
+
+package com.linecorp.armeria.server.docs;
+
+@FunctionalInterface
+public interface DocServiceBuilderSupplier {
```

Comment:
- How about `DocServiceConfigurator`?
- How about implementing `Ordered`, like we did in https://github.com/line/armeria/blob/master/spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/spring/ArmeriaServerConfigurator.java ?
- How about just returning void? Is there a reason you wanted to return a `DocServiceBuilder`?

Reply:
- I'll change the name for `DocServiceConfigurator` I put supplier because of the description in the issue.
- I'll check `Ordered` thanks for the info.
- I wanted to return a `DocServiceBuilder` so it'd be a wrapped `DocServiceBuilder` with the customized methods.